yarn.lock

```diff
@@ -11,10 +11,10 @@
     orbit-camera-controller "^4.0.0"
     turntable-camera-controller "^3.0.0"
 
-"@abi-software/flatmap-viewer@^2.3.3-b.4":
-  version "2.3.3-b.4"
```

Comment:
How has one version number gone backwards and another gone forwards?

Reply:
Looks like it is the automated increment when installing the package on mapintegrated viewer.